tests/providers/test_bank.py

```diff
@@ -136,6 +137,21 @@ def test_iban(self, faker, num_samples):
             assert re.fullmatch(r"\d{2}\d{24}", iban[2:])
 
 
+class TestUkUA:
```

Comment:
The capitalization is actually meaningful here, because of how we run tests.

```suggestion
class TestUkUa:
```